CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1
```

Comment:
I think this should be `1.1.0` since this is a new feature, not a bug fix.

Reply:
Done!

Reply:
(not sure if we should change protocolVersion to 0.2.0 for same reason?)